src/libs/antares/study/parts/thermal/container.cpp

```diff
@@ -148,8 +144,8 @@ uint PartThermal::removeDisabledClusters()
 
     for (auto& it : list)
```

Comment:
About **removeDisabledClusters** : 
Exactly the same code as in **renewable/container.cpp**.
So , we should : 
- Find a way to have one function for both thermal and renewable clusters
- apply remarks about this function in **renewable/container.cpp**.